src/ProjectService.Models.Db/DbProject.cs

```diff
@@ -40,6 +40,14 @@ public void Configure(EntityTypeBuilder<DbProject> builder)
             builder
                 .HasKey(p => p.Id);
 
+            builder
+                .Property(P => P.Name)
+                .IsRequired();
```

Comment:
in 55 line already exist this  condition

Reply:
Fixed